packages/@lwc/compiler/src/options.ts

```diff
@@ -58,6 +58,7 @@ export interface TransformOptions {
     outputConfig?: OutputConfig;
     isExplicitImport?: boolean;
     preserveHtmlComments?: boolean;
+    cssScopeToken: string | undefined;
```

Comment:
I had to revert this, because TypeScript was complaining about some inscrutable type compatibility thing:

```
src/options.ts(144,5): error TS2322: Type '{ stylesheetConfig: RecursiveRequired<StylesheetConfig>; outputConfig: Required<OutputConfig>; experimentalDynamicComponent: Required<...>; ... 4 more ...; cssScopeToken?: string | undefined; }' is not assignable to type 'NormalizedTransformOptions'.
  Types of property 'cssScopeToken' are incompatible.
    Type 'string | undefined' is not assignable to type 'string'.
      Type 'undefined' is not assignable to type 'string'.
```

I don't understand why it treats `foo?: string` differently from `foo: string | undefined`.

Reply:
OK I fixed it by adding `as NormalizedTransformOptions`.